run

```diff
@@ -21,20 +21,22 @@ else
     exit 1
 fi
 
-HEADLESS=1
+HEADLESS=${HEADLESS}
+
 # Set Docker container ID
 container_id=webcentral
 currentDirInContainer=$(docker exec $container_id pwd)
+
+print_usage() {
+  printf "run script: -v <command>"
```

Comment:
For what do we need the print statement?

Reply:
I wanted to add a optional flag to make it possible to run the selenium tests in headless mode, but it doesnt work at the moment. I can remove the code.